persistence-jpa/javadsl/src/main/java/com/lightbend/lagom/javadsl/persistence/jpa/JpaSession.java

```diff
@@ -3,11 +3,14 @@
  */
 package com.lightbend.lagom.javadsl.persistence.jpa;
 
+import akka.Done;
+
 import javax.persistence.EntityManager;
 import java.util.concurrent.CompletionStage;
+import java.util.function.Consumer;
 import java.util.function.Function;
 
 public interface JpaSession {
-    <T> CompletionStage<T> withEntityManager(Function<EntityManager, T> block);
     <T> CompletionStage<T> withTransaction(Function<EntityManager, T> block);
+    CompletionStage<Done> withTransaction(Consumer<EntityManager> block);
```

Comment:
I'll probably give these different names, because trying it out in Chirper I found cases that the compiler considered ambiguous and forced an explicit cast.

I'm thinking `withTransactionReturn` and `withTransactionDo`. Any opinions?